pkgs/development/python-modules/Nikola/fix_markdown_test.patch

```diff
@@ -0,0 +1,25 @@
+From 226e8a726e85525fb22664d9d76f4fe1729cf51d Mon Sep 17 00:00:00 2001
+From: Chris Warrick <kwpolska@gmail.com>
+Date: Fri, 7 Feb 2020 18:00:04 +0100
+Subject: [PATCH] Update tests for Markdown==3.2 output change
+
+This minor change does not affect the appearance of Nikola's output.
+---
+ tests/test_compile_markdown.py | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/tests/test_compile_markdown.py b/tests/test_compile_markdown.py
+index 4d0e3d89c..88ac290d0 100644
+--- a/tests/test_compile_markdown.py
++++ b/tests/test_compile_markdown.py
+@@ -31,8 +31,8 @@
+ <table class="codehilitetable"><tr><td class="linenos">\
+ <div class="linenodiv"><pre>1</pre></div>\
+ </td><td class="code"><pre class="code literal-block"><span></span>\
+-<span class="kn">from</span> <span class="nn">this</span>
+-</pre>
++<code><span class="kn">from</span> <span class="nn">this</span>
++</code></pre>
```

Comment:
there's nothing nix specific here, probably should make a patch against upstream?

Reply:
It's already fixed in upstream (from version 8.1.0 onwards). So an option to prevent including this would be to upgrade to the latest version.

Reply:
another option is to use fetchpatch, here an example: https://github.com/NixOS/nixpkgs/pull/95524/files

Reply:
Seconded - change this to a `fetchpatch` of the relevant upstream commit (for a github commit you can usually just add `.patch` to the end of the url) and I'll review this.